name-registry/src/errors.sw

```diff
@@ -0,0 +1,9 @@
+library errors;
+
+pub enum Errors {
```

Comment:
Can you split this up into different categories?

Reply:
What categories do you suggest?